### PR TITLE
Replace chai React testing docs with Jest

### PR DESCRIPTION
**Pull request by benmvp**

### CODE REVIEW

File: react/testing.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Guidelines and best practices used by Eventbrite to provide consistency and prev
 
 ## Testing environment
 
-Eventbrite uses [`chai`](http://chaijs.com) (`expect` [BDD style](http://chaijs.com/api/bdd/)), [`enzyme`](http://airbnb.io/enzyme/) and [`sinon`](http://sinonjs.org/) for unit testing React components. We also leverage [`chai-enzyme`](https://github.com/producthunt/chai-enzyme) and [`sinon-chai`](https://github.com/domenic/sinon-chai) assertion helpers. Enzyme wraps [`ReactTestUtils`](https://facebook.github.io/react/docs/test-utils.html), which contains a bunch of primitives for testing components. Don't use `ReactTestUtils` directly; use Enzyme!
+Eventbrite uses [Jest](http://facebook.github.io/jest/) and [`enzyme`](http://airbnb.io/enzyme/) for unit testing React components. We also leverage [`jest-enzyme`](https://github.com/blainekasten/enzyme-matchers) assertion helpers. Enzyme wraps [`ReactTestUtils`](https://facebook.github.io/react/docs/test-utils.html), which contains a bunch of primitives for testing components. Don't use `ReactTestUtils` directly; use Enzyme!
```
Comment (gagoar) [inline react/testing.md]: I would use *Don't use `ReactTestUtils` *

File: react/testing.md [hunk 2]
```diff
@@ -71,27 +71,33 @@ The `data-spec` attribute is specific to testing and not tied to presentation li
 </div>
 ```
 
-Tests using `getSingleSpecWrapper` would look like:
+Tests using `getSpecWrapper` would look like:
```
Comment (gagoar) [inline react/testing.md]: so the output of getSpecWrapper is different depending on what we get back? single node or array of nodes?

File: react/testing.md (hunk 1 above)
Comment (BenAtEventbrite) [inline react/testing.md]: 👍

File: react/testing.md (hunk 2 above)
Comment (BenAtEventbrite) [inline react/testing.md]: no it's always an array of nodes. it'll just be an array of 1. We simplified `getSingleSpecWrapper` & `getSpecWrappers` to just `getSpecWrapper`. Having both was confusing & `getSingleSpecWrapper` was a pain to use.

**Pull request merged**